spec.emu

```diff
@@ -161,7 +161,7 @@ contributors: Chengzhong Wu, Hemanth HM, Ecma International
 
       <emu-clause id="sec-errorobjects-install-error-cause">
         <h1><ins>InstallErrorCause ( _O_, _options_ )</ins></h1>
-        <p><ins>The abstract operation InstallErrorCause takes arguments _O_ and _options_. It is used to create `"cause"` property on _O_ when the `"cause"` property is present on _options_. It performs the following steps when called:</ins></p>
+        <p><ins>The abstract operation InstallErrorCause takes arguments _O_ (an Object) and _options_. It is used to create `"cause"` property on _O_ when the `"cause"` property is present on _options_. It performs the following steps when called:</ins></p>
         <emu-alg>
           1. <ins>Assert: Type(_O_) is Object.</ins>
```

Comment:
```suggestion
```
this is now redundant

Reply:
I found it is common to assert the type of arguments in the prose and in the AO algorithms (e.g. https://tc39.es/ecma262/#sec-get-o-p). Is it a convention to omit the assertion step in the algorithms?

Reply:
The main spec is indeed not yet consistent on this, but yes, the intended direction is for prose assertions to *not* be repeated in algorithm steps, and for any algorithm step assertion that can be a prose assertion to become one.